com.unity.netcode.gameobjects/Runtime/Simulator/NetworkSimulationConfiguration.cs

```diff
@@ -0,0 +1,101 @@
+﻿using UnityEngine;
+
+namespace Unity.Netcode
+{
+    [CreateAssetMenu(
+        fileName = nameof(NetworkSimulationConfiguration),
+        menuName = "Multiplayer/" + nameof(NetworkSimulationConfiguration))]
+    public class NetworkSimulationConfiguration : ScriptableObject
+    {
+        /// <summary>
+        /// Network simulation configuration name.
+        /// </summary>
+        [field: SerializeField]
+        public string Name { get; set; }
+
+        /// <summary>
+        /// Optional description of the configuration.
+        /// </summary>
+        [field: SerializeField]
+        public string Description { get; set; }
+
+        /// <summary>
+        /// Value for the delay between packet in milliseconds.
+        /// </summary>
+        [field: SerializeField]
+        public int PacketDelayMs { get; set; }
+
+        /// <summary>
+        /// Value for the network jitter (variance) in milliseconds.
+        /// </summary>
+        [field: SerializeField]
+        public int PacketJitterMs { get; set; }
+
+        /// <summary>
+        /// Value for at which interval packet are dropped
+        /// This value is a drop every X packet, not in time.
+        /// </summary>
+        [field: SerializeField]
+        public int PacketLossInterval { get; set; }
+
+        /// <summary>
+        /// Value for the average percentage of packet are dropped.
+        /// </summary>
+        [field: SerializeField]
+        public int PacketLossPercent { get; set; }
+
+        /// <summary>
+        /// Value for the percentage of packet that should be duplicate.
+        /// </summary>
+        [field: SerializeField]
+        public int PacketDuplicationPercent { get; set; }
+
+        // TODO: Clear up what this do
+        [field: SerializeField]
+        public int PacketFuzzFactor { get; set; }
+
+        // TODO: Clear up what this do
+        [field: SerializeField]
+        public int PacketFuzzOffset { get; set; }
```

Comment:
That's what I thought and I wanted to check with you. Since right now, the simulator is offering it, we will keep it. User needs wants it too. It's not a bad thing to have.

Reply:
We had a chat during last netsim sync and will probably actually remove this.